### PR TITLE
Add keep_days option to recorder.purge_entities

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

https://github.com/home-assistant/core/pull/89726

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/89726
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/recorder.markdown [hunk 1]
```diff
@@ -216,9 +216,28 @@ Call the service `recorder.purge_entities` to start a task that purges events an
 | `entity_id`            | yes<sup>*</sup>      | A list of entity_ids that should be purged from the recorder database. |
 | `domains`               | yes      | A list of domains that should be purged from the recorder database. |
 | `entity_globs`         | yes      | A list of regular expressions that identify entities to purge from the recorder database. |
+| `keep_days`            | yes      | Number of history days to keep in database of matching rows. The default of 0 days will remove all matching rows. |
```
Comment (frenck) [inline source/_integrations/recorder.markdown]:
```suggestion
| `keep_days`            | yes      | Number of history days to keep in the database of matching rows. The default of 0 days will remove all matching rows. |
```

File: source/_integrations/recorder.markdown [hunk 2]
```diff
@@ -216,9 +216,28 @@ Call the service `recorder.purge_entities` to start a task that purges events an
...
 
 Note: The `entity_id` is only optional when used in `automations.yaml` or `scripts.yaml`. When using the UI to call this service then it is mandatory to specify at least one `entity_id` using the Target Picker or via YAML mode.
 
+#### Example automation to remove data rows for specific entities
+
+The below automation will remove history for `sensor.power_sensor_0` older than 5 days at `04:15:00` in every day.
+
+```yaml
+alias: Purge noisy power sensors
```
Comment (frenck) [inline source/_integrations/recorder.markdown]:
```suggestion
alias: "Purge noisy power sensors"
```

File: source/_integrations/recorder.markdown [hunk 3]
```diff
@@ -216,9 +216,28 @@ Call the service `recorder.purge_entities` to start a task that purges events an
 | `entity_id`            | yes<sup>*</sup>      | A list of entity_ids that should be purged from the recorder database. |
 | `domains`               | yes      | A list of domains that should be purged from the recorder database. |
 | `entity_globs`         | yes      | A list of regular expressions that identify entities to purge from the recorder database. |
+| `keep_days`            | yes      | Number of history days to keep in database of matching rows. The default of 0 days will remove all matching rows. |
 
 Note: The `entity_id` is only optional when used in `automations.yaml` or `scripts.yaml`. When using the UI to call this service then it is mandatory to specify at least one `entity_id` using the Target Picker or via YAML mode.
 
+#### Example automation to remove data rows for specific entities
+
+The below automation will remove history for `sensor.power_sensor_0` older than 5 days at `04:15:00` in every day.
```
Comment (frenck) [inline source/_integrations/recorder.markdown]:
```suggestion
The below automation will remove history for `sensor.power_sensor_0` older than 5 days at `04:15:00` every day.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by bdraco** [comment]
thanks

**Pull request merged**